main.go

```diff
@@ -52,6 +55,20 @@ func getConfig() (*config, error) {
 	gitlabToken := os.Getenv("GITLAB_TOKEN")
 	gitlabBaseURL := os.Getenv("GITLAB_BASE_URL")
 	integrationDirectory := os.Getenv("INTEGRATION_DIRECTORY")
+	osToEntGithubToken  := os.Getenv("OS_SYNC_GITHUB_TOKEN")
+	logLevel, found := os.LookupEnv("INTEGRATION_TEST_RUNNER_LOG_LEVEL")
```

Comment:
Lluís from the future says: thanks for adding this! :tada:

Reply:
I take it the name is okay then. Grew quite long :p